### DISCUSSION TITLE
qr code generator index

### DISCUSSION THREAD

**Discussion by jsiver** [gopro/labs#1013] [Q&A]
Does anyone know if there is a single document (or URL) that includes an explanation or description of all the QR code generator fields?  As a new to gopro owner trying to decipher all the abbreviations is nearly impossible. Thanks.

**Comment by dnewman-gpsw** [comment]
Commands are here: https://gopro.github.io/labs/control/tech/
Extensions are here: https://gopro.github.io/labs/control/extensions/

That covers 99% of it.
